impl/src/main/java/com/okta/sdk/impl/oauth2/DPoPInterceptor.java

```diff
@@ -113,7 +129,7 @@ private void processRequest(HttpRequest request, boolean tokenRequest) {
 
     private String getUriWithoutQueryString(HttpRequest request) {
         try {
-            return StringUtils.substringBefore(request.getUri().toString(), "?");
+            return URLDecoder.decode(StringUtils.substringBefore(request.getUri().toString(), "?"), StandardCharsets.UTF_8);
```

Comment:
@arvindkrishnakumar-okta This method is only available since Java 10 => will break Java 8/9 users.